src/courtroom.cpp

```diff
@@ -3762,15 +3762,28 @@ void Courtroom::set_scene(QString f_desk_mod, QString f_side)
   }
 }
 
-void Courtroom::set_self_offset(QString p_list) {
+void Courtroom::set_self_offset(QString p_list, QString p_effect) {
```

Comment:
I am not sure why this method takes in a QString when it should really take in a QStringList.